.github/ISSUE_TEMPLATE/bug_report_form.yml

```diff
@@ -0,0 +1,209 @@
+name: Bug report
+description: File a bug report
+title: "[BUG] "
+labels:
+  - kind/bug
+  - meta/needs triage
+  - priority/medium
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for participating in this project and reporting a bug. Docker Mail Server (DMS) is a community-driven project, and each contribution counts.
+
+        Please **fill out all the fields and checkboxes of this form** to make it easier for maintainers to understand the problem and to solve it. If this template is not adhered to, it will be marked with a label called `meta/no template - no support` and will not be worked on.
```

Comment:
> If this template is not adhered to, it will be marked with a label called `meta/no template - no support` and will not be worked on.

Reading this make me feel pretty bad

And would probably make other people feel bad too about how the issue they are trying to report could be handled

Maybe we could add some kindness with a better phrasing ?

Reply:
I am open for suggestions :)